### PR TITLE
Add `update_title` config option

### PR DESCRIPTION
**Pull request by etpinard**
fixes https://github.com/plotly/Dash.jl/issues/101

- add `update_title` String field to DashConfig struct
- use `"Updating..." as default
- inject its value into the frontend, to make it available to the dash renderer

TODO:

- [x]  should we allow `update_title=nothing` inputs?
- [x] add tests

### CODE REVIEW

File: src/app/dashapp.jl [hunk 1]
```diff
@@ -280,6 +281,12 @@ If a parameter can be set by an environment variable, that is listed as:
 - `compress::Bool`: Default ``true``, controls whether gzip is used to compress
         files and data served by HTTP.jl when supported by the client. Set to
         ``false`` to disable compression completely.
+
+- `update_title::String`: Default ``Updating...``. Configures the document.title
+        (the text that appears in a browser tab) text when a callback is being run.
+        Set to '' if you don't want the document.title to change or if you
+        want to control the document.title through a separate component or
+        clientside callback.
```
Comment (etpinard) [inline src/app/dashapp.jl:289]: mostly copied from 

https://github.com/plotly/dash/blob/edce37426470b900a0081969af30364e43c9a888/dash/dash.py#L243-L247

except for the `None` part.

Comment (alexcjohnson) [reply src/app/dashapp.jl:289]: Only supporting `''` is fine by me.

**Comment by alexcjohnson** [comment]
re: tests - dynamic behavior is covered by the tests in https://github.com/plotly/dash/pull/1315, here it would be fine to just `assert dashjl.driver.execute_script("return store.getState().config.update_title") == expected` for the default and one non-default case.

**Comment by etpinard** [comment]
> here it would be fine to just `assert dashjl.driver.execute_script("return store.getState().config.update_title") == expected` for the default and one non-default case.

My attempt in -> https://github.com/plotly/Dash.jl/pull/102/commits/4b07cab205ded6acd39d8a1c23b9707db58c7069

Thanks for tips!

**Comment by alexcjohnson** [comment]
LGTM! I've enabled forks in CircleCI, didn't realize we hadn't done that yet. Would you push an empty commit to see if the tests will run?

**Comment by alexcjohnson** [comment]
@waralex This step fails for forks:

https://github.com/plotly/Dash.jl/blob/cb64b949144f745908610ed51082d3ffb59fec71/test/ci_prepare.jl#L7

But we already have the appropriate version of the code in a local directory, can't we just use that?

**Comment by waralex** [comment]
> @waralex This step fails for forks:
> 
> https://github.com/plotly/Dash.jl/blob/cb64b949144f745908610ed51082d3ffb59fec71/test/ci_prepare.jl#L7
> 
> But we already have the appropriate version of the code in a local directory, can't we just use that?

Yes, we can use  `Pkg.develop`  instead of  ` Pkg.add`

```
 Pkg.develop(pkg::Union{String, Vector{String}}; io::IO=DEFAULT_IO[])
  Pkg.develop(pkgs::Union{Packagespec, Vector{Packagespec}}; io::IO=DEFAULT_IO[])

  Make a package available for development by tracking it by path. If pkg is given with only a name or by a URL, the package will be downloaded to the location specified by the environment variable JULIA_PKG_DEVDIR, with .julia/dev as the default.

  If pkg is given as a local path, the package at that path will be tracked.

  Examples
  ≡≡≡≡≡≡≡≡≡≡

  # By name
  Pkg.develop("Example")

  # By url
  Pkg.develop(url="https://github.com/JuliaLang/Compat.jl")

  # By path
  Pkg.develop(path="MyJuliaPackages/Package.jl")

  See also PackageSpec
```

**Comment by waralex** [comment]
@alexcjohnson, @etpinard In fact, there is another problem - the components are placed by direct reference to git, and not through the package repository. Apparently I wrote this file when the components were not registered yet. I fixed it in this PR #104. In that #104 unit tests pass, let's check in current one. Perhaps you should just copy the changes from #104 to this PR and merge them within current PR

**Review by alexcjohnson — APPROVED** [review]
Great - Thanks @etpinard ! 💃

**Pull request merged**